### DISCUSSION TITLE
Loading a trillion rows of weather data into TimescaleDB

### DISCUSSION THREAD

**Discussion by ali-ramadhan** [ali-ramadhan/ali-ramadhan.Github.io#2] [General]
[Building a weather data warehouse part I: Loading a trillion rows of weather data into TimescaleDB](https://aliramadhan.me/2024/03/31/trillion-rows.html)

[[Hacker News](https://news.ycombinator.com/item?id=40051191)]

**Comment by JKLOVIE** [comment]
Hi, I want to do the same thing as you, have you considered using Clickhouse database?

**Reply by ali-ramadhan** [reply]
Hi @JKLOVIE sorry I'm just seeing your question now.

I am interested in comparing TimescaleDB with Clickhouse on insert and query performance! I haven't looked into this yet, but I'm hoping to do so in the near future.

Let me know what you end up finding!
